python/ittapi/compat/README.md

```diff
@@ -0,0 +1,59 @@
+# ittapi.compat
+
+The module is a wrapper to an earlier version of Python bindings: ([itt-python](https://github.com/oleksandr-pavlyk/itt-python/)). The idea is to allow users to effortlessly migrate to the `ittapi` package without losing any existing functionality. The module is a wrapper to following ITT APIs:
```

Comment:
```suggestion

The module provides API of an earlier version of Python bindings: ([itt-python](https://github.com/oleksandr-pavlyk/itt-python/)). The idea is to allow users to effortlessly migrate to the `ittapi` package without losing any existing functionality. The module exposes the following ITT APIs:
```

Reply:
I have changed the wording as suggested. Please have a look.